php-checks/src/main/java/org/sonar/php/checks/MultilineBlocksCurlyBracesCheck.java

```diff
@@ -32,35 +32,34 @@
 import org.sonar.plugins.php.api.tree.statement.IfStatementTree;
 import org.sonar.plugins.php.api.tree.statement.StatementTree;
 import org.sonar.plugins.php.api.tree.statement.WhileStatementTree;
+import org.sonar.plugins.php.api.visitors.PreciseIssue;
 
 @Rule(key = MultilineBlocksCurlyBracesCheck.KEY)
 public class MultilineBlocksCurlyBracesCheck extends AbstractStatementsCheck {
 
   public static final String KEY = "S2681";
-  private static final String MESSAGE_LOOP = "Only the first line of this %s-line block will be executed in a loop. The rest will execute only once.";
-  private static final String MESSAGE_IF = "Only the first line of this %s-line block will be executed conditionally. The rest will execute unconditionally.";
+  private static final String MESSAGE_LOOP = "This statement will not be executed in a loop; only the first statement of this %s-statement block will be. The rest will execute only once.";
+  private static final String MESSAGE_IF = "This statement will not be executed conditionally; only the first statement of this %s-statement block will be. The rest will execute unconditionally.";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 196 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)